lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -98,6 +101,40 @@ states:
       next:
         targetState: IDENTITY_START_PAGE
 
+  REPROVE_IDENTITY_START_COI:
+    response:
+      type: page
+      pageId: reprove-identity-start
+      context: coi
+    events:
+      address-only:
+        targetJourney: UPDATE_ADDRESS
+        targetState: START
+      names-dob:
+        targetState: UPDATE_NAME_DOB
+      given-names-only:
+        targetJourney: UPDATE_NAME
+        targetState: UPDATE_NAME_WITHOUT_ADDRESS_START
+      family-name-only:
+        targetJourney: UPDATE_NAME
+        targetState: UPDATE_NAME_WITHOUT_ADDRESS_START
+      given-names-and-address:
+        targetJourney: UPDATE_NAME
+        targetState: UPDATE_NAME_WITH_ADDRESS_START
+      family-name-and-address:
+        targetJourney: UPDATE_NAME
+        targetState: UPDATE_NAME_WITH_ADDRESS_START
+      next:
+        targetState: IDENTITY_START_PAGE
+
+  UPDATE_NAME_DOB:
+    response:
+      type: page
+      pageId: update-name-date-birth
+    events:
+      end:
+        targetState: REPROVE_IDENTITY_START
```

Comment:
should this have a feature flag check for `coiEnabled` and send to `REPROVE_IDENTITY_START_COI` if so?

Reply:
The check is already done in REPROVE_IDENTITY, so should be alright without it here as UPDATE_NAME_DOB can only be reached from it?